app/models/classification.rb

```diff
@@ -1,7 +1,9 @@
 class Classification < ApplicationRecord
-  belongs_to :workflow
```

Comment:
This would probably remain a workflow. It's mostly a temporary copy of the stream event so that we don't have to put the entire JSON into the Sidekiq queue, but just the ID.